### DISCUSSION TITLE
Is QUASINANO2015 parameter set available to use in DFTB?

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1488] [General]
Hello everyone,

I am a new user of DFTB+ and I was searching some good parameter sets for my system. While going through literatures, I found about the QUASINANO2015 parameter sets but I couldn't download those files. Are they suitable to use with DFTB+?
Have someone used this parameter sets so far?

Thank you for your time and consideration!

**Comment by awvwgk** [comment]
The Quasinano parameters are only available in the DFTB module of the Amsterdam modeling suite (AMS) as far as I know. The SK files for electronic energy could be read by DFTB+ in principle, but the repulsion form used in theses SK files is not available in DFTB+. Also, the Quasinano parameters shipped in AMS are only provided as encrypted files, so you would need to extract the encryption key from the AMS binaries before you can load them into DFTB+.
